modules/abstract-eth/src/abstractEthLikeNewCoins.ts

```diff
@@ -70,6 +70,9 @@ import {
   TransactionBuilder,
   TransferBuilder,
 } from './lib';
+import { decode } from 'cbor-x';
+import { isRight } from 'fp-ts/Either';
+import { Buffer } from 'buffer';
```

Comment:
```suggestion
```
I wonder if we could move all this gg18 <> mpcv2 logic to the mpc lib as utils so we dont need to install `cbor-x` and `fp-ts/Either`

Reply:
created utility function to decode ReducedKeyPair and removed direct cbor-x and fp-ts dependency

Reply:
the sdk-lib-mpc should just be the mpc primitives, lets not couple that to bitgo specific encodings.